src/components/container/ConnectDevicesMenu/ConnectDevicesMenu.tsx

```diff
@@ -211,21 +251,28 @@ export default function (props: ConnectDevicesMenuProps) {
 interface AppleHealthMenuItemProps {
     preview: boolean;
     platform: string;
+    connected?: boolean;
+    requested: boolean;
+    enableAppleHealthSurvey?: string;
 }
 
 function AppleHealthMenuItem(props: AppleHealthMenuItemProps) {
     let [expanded, setExpanded] = useState(false);
 
-    if (props.platform == "Android") {
-        return null;
-    }
-
     let action = () => setExpanded(!expanded);
-    let indicator = <div className="mdhui-connect-devices-menu-connect">{language("how-to-enable")}</div>;
+    let indicator = <div className="mdhui-connect-devices-menu-connect">{language(props.connected ? "connected" : "how-to-enable")}</div>;
+
+    if (props.enableAppleHealthSurvey && !props.requested) {
+        action = () => {
+            if (props.preview) return;
+            MyDataHelps.startSurvey(props.enableAppleHealthSurvey!);
+        }
+        indicator = <div className="mdhui-connect-devices-menu-connect">{language("connect")}</div>;
+    }
 
     if (props.platform == "Web") {
```

Comment:
Maybe something similar here to avoid unnecessarily creating and overriding the actions and indicators.